xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -558,17 +558,20 @@ void CLinuxRendererGL::DrawBlackBars()
 
   glUniform4f(uniCol, m_clearColour / 255.0f, m_clearColour / 255.0f, m_clearColour / 255.0f, 1.0f);
 
+  int iScreenWidth = CServiceBroker::GetWinSystem()->GetGfxContext().GetWidth();
```

Comment:
"screen" is not the right term here. Kodi's main window might not be fullscreen.

bzw: for windowed mode the area outside of viewRect is always visible.

Reply:
iWindowWidth okay with you?

Reply:
something line osWindowWidth/Height is less ambiguous